node/benchmarks/index.js

```diff
@@ -20,21 +20,28 @@
 
 'use strict';
 
+/*eslint no-console: 0*/
```

Comment:
we're not yet using that linter, let's not just start adding habitual exceptions until we do... plus won't it complain about the comment formatting here?